src/test/emulators/functionsRuntimeWorker.spec.ts

```diff
@@ -92,6 +93,7 @@ class MockRuntimeBundle implements FunctionsRuntimeBundle {
   triggerType = EmulatedTriggerType.HTTPS;
   cwd = "/home/users/dir";
   ports = {};
+  socketPath = "";
```

Comment:
nit: Please just initialize to undefined if it is supposed to be unspecified initially. Empty string does not give us much value here, takes more effort to check, and takes away a lot of value from type checking.